src/ui-components/error-state/error-state.component.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import { Tile } from "carbon-components-react";
+import { Trans, useTranslation } from "react-i18next";
+
+import styles from "./error-state.scss";
+import ErrorIllustration from "./error-illustration.component";
+
+const EmptyState: React.FC<ErrorStateProps> = props => {
+  const { t } = useTranslation();
+
+  return (
+    <Tile light>
+      <h1 className={styles.heading}>{props.headerTitle}</h1>
+      <ErrorIllustration />
+      <p className={styles.content}>
+        <Trans i18nKey="errorStateText">Sorry, there was an error</Trans>
+      </p>
+      <p className={styles.errMessage}>Database does not exist for this tile</p>
```

Comment:
This message should be a prop, right?

Reply:
Indeed it should. I left it as so pending clarification about exactly where this message should come from and exactly what it should look like. See PR description above for a bit on Ciaran's thoughts.